activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -175,9 +175,6 @@ def test_reconnection_after_actual_disconnection_with_verify
       assert_not_equal original_connection_pid, new_connection_pid,
         "umm -- looks like you didn't break the connection, because we're still " +
         "successfully querying with the same connection pid."
-
-      # Repair all fixture connections so other tests won't break.
-      @fixture_connections.each(&:verify!)
```

Comment:
These lines were redundant since an actual connection that was broken is repaired a few lines before.